controllers/link.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+const api = require('../api');
+
+router.post('/', function (req, res, next) {
+    let target = `${req.protocol}://${req.get('host')}/register/${req.body.schoolId}`;
```

Comment:
@MasterCarl changed it in new commit to `req.headers.origin`, worked on production and staging in https://github.com/schul-cloud/schulcloud-client/blob/master/controllers/pwrecovery.js#L12 😄